_includes/footer.html

```diff
@@ -7,7 +7,7 @@
             <span>
                 {% for item in site.data.socialMedia %}
                 <a href="{{ item.link }}" class="social-item">
-                    <img src="{{ item.source | relative_url }}" alt="{{ item.name | capitalize }} Logo">
+                    {% include {{ item.source }} %}
```

Comment:
You can "include" svgs like regular partials, this will make the SVGs inline, allowing us to style them and much more.

Reply:
Cool, but I would like to keep the alt text capitalized anyway, thanks a lot